dojo/jira_link/helper.py

```diff
@@ -535,6 +535,12 @@ def get_tags(obj):
         if obj_tags:
             for tag in obj_tags:
                 tags.append(str(tag.name))
+    if type(obj) == Finding_Group:
+        for finding in obj.findings.all():
+            obj_tags = finding.tags.all()
+            if obj_tags:
+                for tag in obj_tags:
+                    tags.append(str(tag.name))
```

Comment:
Can this be updated to remove potential duplicates in the list

Reply:
@schuman0 I was talking about the finding groups. Findings and engagements already cannot have the same tag. If a Finding A and Finding B have the same tag, this code will put two of the same tag in the same list and attempt to push to jira.

Reply:
My apologies. I added it in the wrong location